extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii

```diff
@@ -28,15 +28,18 @@ tags: {  # amp-lightbox-gallery
 attr_lists: {
   name: "lightboxable-elements"
   attrs: { name: "lightbox" }
+  attrs: {
+    name: "lightbox-thumbnail-id"
+    value_regex_casei: "^[a-z][a-z\\d_-]*"
+  }
+}
+attr_lists: {
```

Comment:
since this is used by amp-carousel, shouldn't this list be in the amp-carousel protoascii?